src/app/pages/quickMaps/components/sideNavContent/sideNavContent.component.scss

```diff
@@ -160,4 +176,42 @@ h3 {
 
 ::ng-deep .mat-select-panel.allow-long-text {
   max-width: unset;
+}
+
+mat-slide-toggle {
+  position: absolute;
+  left: 0em;
+  top: 0em;
+}
+
+::ng-deep {
```

Comment:
Using ::ng-deep without nesting isn't good.
You could use a more exact "matTooltipClass" in the template then target the tooltip using "::ng-deep .quickmaps-sidebar-lock-tooltip", for example.

To target the toggle you should put a css class on it ("sidebar-lock-toggle") then target using ".sidebar-lock-toggle ::ng-deep blahhh".